vortexor/Readme.md

```diff
@@ -145,3 +145,92 @@ Operators can decide whether to adopt Vortexors without concerns about network
 protocol changes. Upgrading involves specifying the Vortexor's TPU address and
 verified packet receiver network address via CLI or Admin RPC. The transition is
 designed to be seamless for operators.
+
+# Pair a Vortexor and Validator
+
+To pair a validator and Vortexor, follow these steps:
+
+---
+
+### Step 1: Run the Vortexor
+Run the Vortexor using the following command:
+
+```bash
+solana-vortexor --identity /path/to/id.json \
+    --destination <validator_receiver_address> \
+    --dynamic-port-range <port_range> \
+    --rpc-server <rpc_server_address> \
+    --websocket-server <websocket_server_address>
+```
+
+**Parameters:**
+- `--identity`: Path to the identity keypair file for the Vortexor.
+- `--destination`: The validator's receiver address where verified packets will be sent (e.g., `10.138.0.136:8100`).
+- `--dynamic-port-range`: The port range used by the Vortexor for TPU traffic (e.g., `9200-9300`).
+- `--rpc-server`: The RPC server address to fetch cluster information (e.g., `http://10.138.0.137:8899`).
```

Comment:
would this normally point to the validator's own RPC?

Reply:
Leader node is normally not running RPC service itself due to load. So normally it would pick RPC nodes other their the paring validator.